content/en/docs/onboarding/01-starting-out.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Section 1: Starting Out"
+type: reveal
+weight: 1
+description: |
+    Begin your journey to become a contributing member of the Kubernetes project!
+---
+
+# Welcome to Kubernetes!
+
+Welcome to the E-Learning for Contributors course for Kubernetes!
+
+---
+
+# Section 1: Starting Out
+
+Each unit of this course consists of a slideshow and links to resources. Take your time reading the materials, and feel free to reach out to the community with questions.
+
+We look forward to your contributions!
+
+---
+
+# What you are about to learn
+
+This first unit is all about starting out in the Kubernetes community. By the end of this unit, you will:
+
+* Understand the organization of the K8s community, special interest groups, committees, and working groups.
+* Communicate effectively with the community.
+* Understand and be able to work with the K8s code of conduct and community values.
+
+---
+
+# Who exactly is a contributor?
+
+Kubernetes contributors bring a wide range of talents, skills, services, and ideas to the community. They:
+
+* Write documentation
+* Write and review code
+* Participate in the community
+* Participate in special interest groups
+* AND MORE!
+
+Everybody's contributions are welcome in the Kubernetes community!
+
+---
+
+# What are the core values of the Kubernetes community?
+
+The Kubernetes community adheres to the following principles:
+
+* Distribution is better than centralization 
+* Community over product or company
+* Automation over process
+* Inclusive is better than exclusive
+* Evolution is better than stagnation
+
+[Read more about our community values.](https://www.kubernetes.dev/community/values/)
```

Comment:
```suggestion
[Read more about our community values.](/community/values/)
```
This can be a find replace of `https://www.kubernetes.dev/` with `/`
across all the changed files

Reply:
Done!